core/src/main/java/io/questdb/cairo/ColumnType.java

```diff
@@ -89,6 +89,7 @@ public final class ColumnType {
     public static final short SYMBOL = 12;
     public static final short TIMESTAMP = 8;
     public static final short UNDEFINED = 0;
+    public static final short UUID = 19;
```

Comment:
as discussed - this would trade code duplication (often in the form of fall-through in switch/case statements) for extra branching when Long128 and UUID are supposed to behave differently.